developer/installation/installation-linux.md

```diff
@@ -37,7 +37,7 @@ curl https://install.meteor.com/ | sh
 
 ```sh
 # install CLI
-npm install -g reaction-cli
+sudo npm install -g reaction-cli
```

Comment:
This one's a little less clear because it all depends on how you've installed node.  If you've installed to /usr/local then presumably you're going to need to sudo.  If you've installed to your unprivileged user's home dir and done something like `export PATH=$PATH:~/node-v8.9.4-linux-x64/bin/`, then the sudo will fail.  Do you all have a preference how to install node?  It looks like the [docker container](https://github.com/reactioncommerce/base/blob/master/scripts/install-node.sh#L14) installs it to /opt, which would mean the sudo here is required.

Reply:
I agree with the sentiment but I don't think just changing this to `sudo` is the solution. The problem is that it differs from distro to distro so there isn't one solution, but generally you shouldn't need to install `reaction-cil` with `sudo`

Reply:
Thanks @zenweasel .  I thought the `-g` flag meant that npm would try to install into the global (system-wide) modules directory, which in most cases is outside of a user's home directory.  I think we cannot make any assumptions here when we've not instructed how to install Node.js so I'll remove the change I've made to this line.